packages/react-google-maps-api/src/components/dom/OverlayView.tsx

```diff
@@ -44,14 +69,69 @@ export interface OverlayViewProps {
   children?: ReactNode | undefined
   // required
   mapPaneName: PaneNames
-  getPixelPositionOffset?: ((offsetWidth: number, offsetHeight: number) => { x: number; y: number }) | undefined
-  bounds?: google.maps.LatLngBounds | google.maps.LatLngBoundsLiteral | undefined
-  position?: google.maps.LatLng | google.maps.LatLngLiteral | undefined
+  position: google.maps.LatLng | google.maps.LatLngLiteral
+  getPixelPositionOffset?:
+    | ((offsetWidth: number, offsetHeight: number) => { x: number; y: number })
+    | undefined
+  bounds?:
+    | google.maps.LatLngBounds
+    | google.maps.LatLngBoundsLiteral
+    | undefined
+  zIndex?: number
   onLoad?: ((overlayView: google.maps.OverlayView) => void) | undefined
   onUnmount?: ((overlayView: google.maps.OverlayView) => void) | undefined
 }
+export const FLOAT_PANE: PaneNames = `floatPane`
+export const MAP_PANE: PaneNames = `mapPane`
+export const MARKER_LAYER: PaneNames = `markerLayer`
+export const OVERLAY_LAYER: PaneNames = `overlayLayer`
+export const OVERLAY_MOUSE_TARGET: PaneNames = `overlayMouseTarget`
+export function OverlayViewF({
```

Comment:
please use memo for export like other F components

Reply:
added memo